pkg/manifests/manifests.go

```diff
@@ -16,6 +16,9 @@ package manifests
 
 import (
 	"bytes"
+	"io/ioutil"
+	"path"
```

Comment:
(nit)
```suggestion
	"path/filepath"
```

Reply:
This is automatically added by vscode :shrug:

Reply:
Agreed, as per the msg  https://golang.org/pkg/path/ 
> This package does not deal with Windows paths with drive letters or backslashes; to manipulate operating system paths, use the path/filepath package.

So this would be more appropriate here.

Reply:
thanks for the detailed explanation :+1: fixed